src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -5368,6 +5377,9 @@ int RGWRados::restore_obj_from_cloud(RGWLCCloudTierCtx& tier_ctx,
       attrs[RGW_ATTR_CLOUD_TIER_CONFIG] = t_tier;
     }
 
+    // TODO : check whether log_op need to passed to this function
```

Comment:
nit: could you please update the comment here on why log_op is set to false for future reference.